app/models/setting/content.rb

```diff
@@ -26,7 +26,8 @@ def self.load_defaults
         self.set('pulp_client_key', N_("Path for ssl key used for pulp server auth"), "/etc/pki/katello/private/pulp-client.key"),
         self.set('pulp_client_cert', N_("Path for ssl cert used for pulp server auth"), "/etc/pki/katello/certs/pulp-client.crt"),
         self.set('remote_execution_by_default', N_("If set to true, use the remote execution over katello-agent for remote actions"), false),
-        self.set('use_pulp_oauth', N_("use oauth authentication for pulp instead of the default cert based authentication"), false)
+        self.set('use_pulp_oauth', N_("use oauth authentication for pulp instead of the default cert based authentication"), false),
+        self.set('unregister_delete_host', N_("when unregistering with subscription-manager delete its Host"), false)
```

Comment:
"it" is used without prior definition, maybe put

"When unregistering host via subscription-manager, also delete server-side host record"